app/api_topologies.go

```diff
@@ -224,7 +244,8 @@ type APITopologyOption struct {
 	Value string `json:"value"`
 	Label string `json:"label"`
 
-	filter render.FilterFunc
+	filter       render.FilterFunc
+	filterPseudo bool
```

Comment:
Do we really need to `filterPseudo` field? Can't we do it including it implicitly in `render.FilterFunc`? (through function composition maybe)